pkg/machines/scripts/get_libvirt_service_name.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+NAME="`systemctl --no-legend list-unit-files libvirtd.service libvirt-bin.service |  head -n1 | cut -f1 -d' '`"
+
+if [ -n "$NAME" ]; then
+    # get id name because libvirt-bin is primary in ubuntu 1604
+    systemctl  --property=Id show "$NAME" | cut -c 4-
+fi
```

Comment:
Again, this isn't necessary. On Ubuntu-16.04, the above command already gives you `libvirt-bin.service`. (But see comment above, I think this file could be eliminated completely).

Reply:
It can also give `libvirtd.service` on Ubuntu 16.04. `libvirt-bin.service` creates alias to `libvirtd.service` when enabled. To make things more interesting we even cannot prefer `libvirt-bin.service` because on ubuntu 17.10 `libvirtd.service` is primary and creates symlink to `libvirt-bin.service`. 

We really need the id of the service, because that is the only service visible when disabled and stopped.

Reply:
"We really need the id of the service" → What does that mean? How is it in any way different than the `$NAME` you already determined? (It's demonstrably not on fedora-27 and ubuntu-1604; I didn't try other cases)

Reply:
I am sorry. It works on ubuntu 16.04 (didn't try this exact command and assumed wrong sort order of those two names). 
But it still breaks on ubuntu 17.10

```
systemctl enable libvirtd.service
systemctl --no-legend list-unit-files libvirtd.service libvirt-bin.service |  head -n1 | cut -f1 -d' '
# libvirt-bin.service
systemctl disable libvirtd.service
systemctl --no-legend list-unit-files libvirtd.service libvirt-bin.service |  head -n1 | cut -f1 -d' '
# libvirtd.service
```

Reply:
appending to the last comment:
```
systemctl status libvirt-bin
# Unit libvirt-bin.service could not be found.
```